contracts/carbon/CarbonController.sol

```diff
@@ -22,18 +21,18 @@ contract CarbonController is
     Strategies,
     Upgradeable,
     ReentrancyGuardUpgradeable,
-    PausableUpgradeable,
     OnlyProxyDelegate,
     Utils
 {
-    // the emergency manager role is required to pause/unpause
-    bytes32 private constant ROLE_EMERGENCY_STOPPER = keccak256("ROLE_EMERGENCY_STOPPER");
-
     // the fees manager role is required to withdraw fees
     bytes32 private constant ROLE_FEES_MANAGER = keccak256("ROLE_FEES_MANAGER");
 
     uint16 private constant CONTROLLER_TYPE = 1;
 
+    // deprecated parent storage vars
+    bool private _deprecated;
+    uint256[49] private __deprecated;
```

Comment:
Why not just `uint256[MAX_GAP] private __deprecated;`?
Also, for readability, probably better placed after all constant and immutable variables.

Reply:
Because I wanted to keep the exact definition from the removed contract